datafusion/catalog/Cargo.toml

```diff
@@ -17,6 +17,7 @@
 
 [package]
 name = "datafusion-catalog"
+description = "datafusion-catalog"
```

Comment:
I had to add a description so that I could publish the crate. It would obviously be better to have a more meaningful description here, but I am not familiar with this new crate yet.